src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/Obsolete/ObsoleteControls.cs

```diff
@@ -0,0 +1,228 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Data;
+using System.Drawing;
+#pragma warning disable WFDEV006, WFDEV007, WFDEV009, WFDEV011, WFDEV015, WFDEV017, WFDEV022, WFDEV025 // Type or member is obsolete
+using static System.Windows.Forms.DataGrid;
+
+namespace WinFormsControlsTest;
+
+/// <summary>
+/// This is added to test compile time compatibility only. 
+/// </summary>
+// Obsolete controls test for https://github.com/dotnet/winforms/issues/3783
+public partial class ObsoleteControls : Form
+{
+    private bool _tablesAlreadyAdded;
+
+    public ObsoleteControls()
+    {
+        try
+        {
+            InitializeComponent();
+            CreateMainMenu();
+            SetUp(); // This code is only valid when redirected to NET48.
+        }
+        catch (PlatformNotSupportedException ex)
+        {
+            MessageBox.Show(ex.Message);
```

Comment:
This is an extra pop-up box that we have to close , this is an extra click when testing. Consider
```suggestion
            title = ex.Message;
```